WordPress/Classes/ViewRelated/NUX/SiteCreationThemeSelectionViewController.swift

```diff
@@ -189,47 +133,41 @@ class ThemeSelectionViewController: UICollectionViewController, LoginWithLogoAnd
         themesSyncHelper?.syncContent()
     }
 
-    private func syncThemePage(_ page: NSInteger, success: ((_ hasMore: Bool) -> Void)?, failure: ((_ error: NSError) -> Void)?) {
-        assert(page > 0)
-        let account = AccountService(managedObjectContext: ContextManager.sharedInstance().mainContext).defaultWordPressComAccount()
+    private func syncThemePage(page: NSInteger, success: ((_ hasMore: Bool) -> Void)?, failure: ((_ error: NSError) -> Void)?) {
 
-        // TODO: replace with new endpoint.
+        guard let siteType = siteType else {
+            return
+        }
 
-        _ = themeService.getThemesFor(account,
-                                      page: page,
-                                      success: { (_, _, _) in
+        _ = themeService.getStartingThemes(forCategory: siteType.rawValue,
```

Comment:
Since this is a new method only used here, and the `NSProgress` is just being tossed away in `_` what do you think of making `getStartingThemesForCategory` return `void` instead?

Reply:
Sure. I just made it consistent with similar methods just in case, but yeppa I can change it.